src/app/page.tsx

```
@@ -1,5 +1,10 @@
-const page = () => {
-  return <p className="head-1 text-error-primary">Glit</p>;
+const Page = () => {
+
+  return (
```

ESLint

[error] Delete `⏎`

(prettier/prettier)